.gitlab-ci.yml

```diff
@@ -128,7 +128,7 @@ test_macos11_py37_coremltools_test:
   dependencies:
     - build_wheel_macos_py37
   variables:
-    WHEEL_PATH: build/dist/*cp37*10_16*
+    WHEEL_PATH: build/dist/*cp37*10_15*
```

Comment:
Is this change intentional?

Reply:
Yes, it’s intentional. We are now only producing one wheel per Python version on macOS. The file name of the wheel contains the min macOS version it supports (i.e. 10.15).